.github/workflows/tests.yml

```diff
@@ -1,7 +1,7 @@
 name: CI
 on:
   push:
-    branches: master
+    branches: [master]
```

Comment:
Why is this needed?

Reply:
It's not really, I had to add my `test` branch so I could check the CI without affecting this PR. Obviously the branch is now removed from the YAML but I left the array markers. I can remove those if you want, but either way the YAML is valid.